docker_image_size_limit/__init__.py

```diff
@@ -126,4 +134,18 @@ def _parse_args() -> argparse.Namespace:
         help='Maximum number of image layers',
         default=-1,
     )
+    parser.add_argument(
+        '--current', '-c',
+        action='store_true',
+        help='Display the current size of the Docker image',
+        default=False,
+        dest='current_size'
+    )
+    parser.add_argument(
+        '--dry-run',
```

Comment:
```suggestion
        '--exit-zero',
```

Reply:
Done in https://github.com/wemake-services/docker-image-size-limit/commit/b76c142f89ffe291e1c800b1daa2c6ebf46a2e4f